core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -107,6 +107,14 @@ class SparkHadoopUtil extends Logging {
         if (key.startsWith("spark.hadoop.")) {
```

Comment:
the comment above does not apply for the whole loop anymore and should be moved to this if statement